src/components/views/elements/Field.tsx

```diff
@@ -242,10 +244,10 @@ export default class Field extends React.PureComponent<PropShapes, IState> {
         const Tooltip = sdk.getComponent("elements.Tooltip");
         let fieldTooltip;
         if (tooltipContent || this.state.feedback) {
-            const addlClassName = tooltipClassName ? tooltipClassName : '';
+            const addClassName = tooltipClassName ? tooltipClassName : '';
```

Comment:
I think `addl` here meant additional. Maybe let's avoid the naming question entirely and use the `classNames` function we use elsewhere for this...?

Reply:
Oh, I thought it must have been a typo I introduced. Sure, let's use `classnames`